app/controllers/reviews_controller.rb

```diff
@@ -4,6 +4,8 @@ class ReviewsController < ApplicationController
   def create
     @review = Review.new(strong_params)
     @video = Video.find(params[:video_id])
+    @review.video = @video
+    @review.user = current_user
```

Comment:
```ruby
@review = Review.new(strong_params)
@video = Video.find(params[:video_id])
@review.video = @video
@review.user = current_user
```

Instead of having 4 lines, you could write this like:

```ruby
@video = Video.find(params[:video_id])
@review = @video.reviews.build(strong_params.merge!(user: current_user))
```